spec/game_spec.rb

```diff
@@ -6,4 +6,21 @@
         pass = true
         expect(pass).to eq(true)
    end
+
+   it "prints a 3x3 board to the console" do 
+      g = Game.new()
+      expect do 
+      g.print_board()
+      end.to output("
+             |       | 
+         1   |   2   |   3
+       _____ | _____ | _____
+             |       |  
+         4   |   5   |   6
+       _____ | _____ | _____
+             |       | 
+         7   |   8   |   9
+             |       | 
+        \n").to_stdout
+   end
```

Comment:
Suppose we decided to have a web app instead of a console app, this test would fail.  This test depends on stdout. 
 How can you structure your classes, so you can test, independent of stdout or any output method?